multinode-infra-app.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+- hosts: localhost
+  tasks:    
+    - name: Show Playbook Directory
+      debug:
+       msg: "{{ playbook_dir }}"
+
+    - name: Template Terraform files into project directory
+      template:
+        src: terraform.tfvars.j2
+        dest: "{{ playbook_dir }}/terraform.tfvars"
+  
+    - name: Template Terraform userdata.cfg.tpl files into project template directory
+      template:
+        src: "{{ playbook_dir }}/templates/userdata.cfg.tpl.j2"
+        dest: "{{ playbook_dir }}/templates/userdata.cfg.tpl"
+
+# Provision the infrastructure The CaaS puts hosts for accessing the OpenStack
+# API into the 'openstack' group
+- hosts: openstack
+  roles:
+    - cluster_infra
+
+- hosts: localhost
+  tasks: 
+  # Check whether an ans_vlt_pwd variable is defined and if so, save it into a
+  # file called '~/vault.password'. If it doesn't exist, create a the
+  # '~/vault.password' file with ans_vlt_pwd = "password_not_set" as the
+  # password.
+    - name: Create vault password file
+      vars:
+        ans_dflt: 'default_password'
```

Comment:
I don't think it makes sense to have a default here.

Reply:
Has been removed all together.